AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Storage/CascadeDeleteOperationTests.swift

```diff
@@ -90,6 +92,47 @@ class CascadeDeleteOperationTests: StorageEngineTestsBase {
         XCTAssertEqual(queriedRestaurants.count, 0)
     }
 
+    func testWithCompositeIdentifier() {
+        let modelId = "model-id"
+        let modelDob = Temporal.DateTime.now()
+        let model = ModelCompositePk(id: modelId, dob: modelDob, name: "name")
+        guard case .success = saveModelSynchronous(model: model) else {
+            XCTFail("Failed to save")
+            return
+        }
+        let predicate: QueryPredicate = ModelCompositePk.keys.id == modelId
+            && ModelCompositePk.keys.dob == modelDob
+        guard case .success(let queriedRestaurants) = queryModelSynchronous(modelType: ModelCompositePk.self,
+                                                                            predicate: predicate) else {
+            XCTFail("Failed to query")
+            return
+        }
+        XCTAssertEqual(queriedRestaurants.count, 1)
```

Comment:
nit: can add assertions on the fields of `model`

Reply:
good call, updated with assertions and renamed to `querieModel`